lib/EmitVerilog/EmitVerilog.cpp

```diff
@@ -739,32 +739,54 @@ class ExprEmitter : public ExprVisitor<ExprEmitter, SubExprInfo>,
   SubExprInfo emitBitSelect(Value operand, unsigned hiBit, unsigned loBit);
 
   SubExprInfo emitBinary(Operation *op, VerilogPrecedence prec,
-                         const char *syntax, bool hasStrictSign = false,
+                         const char *syntax,
```

Comment:
I think it would be best to split the yosis integration work out from the behavior changes here.  These are logically distinct changes.